apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -471,8 +473,9 @@ export class ApiModelGenerator {
       const apiItemMetadata: ApiItemMetadata = this._collector.fetchApiItemMetadata(astDeclaration);
       const docComment: tsdoc.DocComment | undefined = apiItemMetadata.tsdocComment;
       const releaseTag: ReleaseTag = apiItemMetadata.effectiveReleaseTag;
+      const isReadonly: boolean = this._determineReadonly(astDeclaration);
 
-      apiEnum = new ApiEnum({ name, docComment, releaseTag, excerptTokens });
+      apiEnum = new ApiEnum({ name, docComment, releaseTag, excerptTokens, isReadonly });
```

Comment:
I don't believe enums can be readonly or not, so remove this? The `const enum` doesn't mean that enum is "readonly" (https://www.typescriptlang.org/docs/handbook/enums.html#const-enums). Note that we'll also then want to remove the mixin from the `ApiEnum` item.